src/main/java/com/aws/greengrass/status/FleetStatusService.java

```diff
@@ -165,15 +163,18 @@ public FleetStatusService(Topics topics, MqttClient mqttClient, DeploymentStatus
             return;
         }
 
-        topics.lookup(CONFIGURATION_CONFIG_KEY, FLEET_STATUS_PERIODIC_UPDATE_INTERVAL_SEC)
-                .dflt(DEFAULT_PERIODIC_UPDATE_INTERVAL_SEC)
+        deviceConfiguration.getPeriodicStatusPublishIntervalTopic()
+                .dflt(DEFAULT_PERIODIC_PUBLISH_INTERVAL_SEC)
                 .subscribe((why, newv) -> {
                     int newPeriodicUpdateIntervalSec = Coerce.toInt(newv);
                     // Do not update the scheduled interval if it is less than the default.
-                    if (newPeriodicUpdateIntervalSec < DEFAULT_PERIODIC_UPDATE_INTERVAL_SEC) {
+                    if (newPeriodicUpdateIntervalSec < DEFAULT_PERIODIC_PUBLISH_INTERVAL_SEC) {
+                        logger.atWarn().log("Not updating periodic publish interval for Fleet Status Service to "
+                                        + "{} because it is less than the minimum default: {}",
+                                newPeriodicUpdateIntervalSec, DEFAULT_FLEET_STATUS_SERVICE_PUBLISH_TOPIC);
```

Comment:
Do we need to reset to DEFAULT_PERIODIC_PUBLISH_INTERVAL_SEC, if the current periodicPublishIntervalSec is longer than DEFAULT_PERIODIC_PUBLISH_INTERVAL_SEC?

Reply:
No. We should keep the same interval as before. This is the same logic we have for telemetry. If the customer wants to set it to default, they can.

Reply:
Then we need to persist the actual interval somewhere? especially in the restart case

Reply:
Discussed offline. This scenario is handled and that we will use the value in the device configuration on start of fresh nucleus or on a restart.